### DISCUSSION TITLE
Volume Plus and Minus buttons not working.

### DISCUSSION THREAD

**Discussion by phreatic62** [rern/rAudio#655] [Fixed Issues] [answered]
Volume + and - buttons do not work after latest update. Dial seems ok, as does the 'mute' button between + and -.
This is the same on my phone browser (touch screen) and my PC browser using the mouse.

Edit: Played around a bit and both DO work, but on press or click and hold, but you can't 'stab' them to go up or down one unit at a time.

**Comment by Raitsa** [comment]
Long press is now needed for - and + volumes.
Before that was not needed, I think this is a fresh bug.

**Comment by phreatic62** [comment, marked as answer]
This issue seems resolved with release 20211126.

Thank you for the rapid response.

**Comment by Raitsa** [comment]
Yeah this is now fixed, thanks a lot @rern
